Added info regarding the name tag (#5459)
cyberbotics/webots: docs/reference/accelerometer.md

Derived from [Device](device.md) and [Solid](solid.md).

### Description

The [Accelerometer](#accelerometer) node can be used to model accelerometer devices such as those commonly found in mobile electronics, robots and game input devices.
The [Accelerometer](#accelerometer) node measures acceleration and gravity induced reaction forces over 1, 2 or 3 axes.
It can be used for example to detect fall, the up/down direction, etc.
The parent node of an [Accelerometer](#accelerometer) node should have a [Physics](physics.md) node defined in its `physics` field, so that correct measurements can be performed.

### Field Summary

```
Accelerometer {

>   SFString name      "accelerometer"  # used by wb_robot_get_device()

Reviewer: Missing space here:
```suggestion
  SFString name       "accelerometer"  # used by wb_robot_get_device()
```

>   MFVec3f lookupTable [ ]    # lookup table
>   SFBool  xAxis       TRUE   # {TRUE, FALSE}
>   SFBool  yAxis       TRUE   # {TRUE, FALSE}
>   SFBool  zAxis       TRUE   # {TRUE, FALSE}

Reviewer: Alignment of comments:
```suggestion
  MFVec3f lookupTable [ ]             # lookup table
  SFBool  xAxis       TRUE            # {TRUE, FALSE}
  SFBool  yAxis       TRUE            # {TRUE, FALSE}
  SFBool  zAxis       TRUE            # {TRUE, FALSE}
```

>   SFFloat resolution  -1     # [0, inf)  

Reviewer: ```suggestion
  SFFloat resolution  -1              # [0, inf)
```

}
```

> **Note**: The description above is a just a partial list of the fields in Accelerometer. The complete list can be found at: [Accelerometer.wrl](https://github.com/cyberbotics/webots/blob/master/resources/nodes/Accelerometer.wrl).

Reviewer: ```suggestion
**Note**: The above description lists only the fields specific to the Accelerometer node. The complete field list can be found in the [Accelerometer.wrl](https://github.com/cyberbotics/webots/blob/master/resources/nodes/Accelerometer.wrl) definition file.
```

> - `name`: This optional field defines the string name used by `wb_robot_get_device()` to get the `WbDeviceTag` to reference this sensor. If not defined in the `.proto` file, the default value described in the `.wrl` file from above is used.

Reviewer: ```suggestion
- `name`: This field defines the string name used by `wb_robot_get_device()` to get the `WbDeviceTag` handle of this sensor. Its default value is "accelerometer".
```

### Field Summary

```
Accelerometer {
  SFString name       "accelerometer" # used by wb_robot_get_device()
  MFVec3f lookupTable [ ]             # lookup table
  SFBool  xAxis       TRUE            # {TRUE, FALSE}
  SFBool  yAxis       TRUE            # {TRUE, FALSE}
  SFBool  zAxis       TRUE            # {TRUE, FALSE}
  SFFloat resolution  -1              # [0, inf)
}
```

> **Note**: The above description lists only the fields specific to the Accelerometer node. The complete field list can be found in the [Accelerometer.wrl](https://github.com/cyberbotics/webots/blob/master/resources/nodes/Accelerometer.wrl) definition file.

Reviewer: Sorry, I missed this one in my first review. This is a better way to link to the correct version of the source code:
```suggestion
**Note**: The above description lists only the fields specific to the Accelerometer node. The complete field list can be found in the [Accelerometer.wrl]({{ url.github_tree }}/resources/nodes/Accelerometer.wrl) definition file.
```

Author: Nice, committing it
